recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+build:
+  number: 0
+  skip: true  # [not win]
```

Comment:
Sure, it does, but couldn't that be desirable?

For instance, I have an old Windows computer that I would like to apply this patch to so that I can build 64-bit VS 2008 binaries with. Is there any reason I should not do that?